KWD/GPIO/src/GPIOKeywordDetector.cpp

```diff
@@ -344,23 +346,23 @@ void GPIOKeywordDetector::detectionLoop() {
             ACSDK_DEBUG0(LX("detectionLoopControlCommand").d("time (us)", std::chrono::duration_cast<std::chrono::microseconds> (end - begin).count() ));
 
             // Read indexes
-            uint64_t current_device_index = readIndex(payload, 1);
-            uint64_t begin_device_index = readIndex(payload, 9);
-            uint64_t end_device_index = readIndex(payload, 17);
-            auto begin_server_index = current_index - (current_device_index - begin_device_index);
+            uint64_t currentDeviceIndex = readIndex(payload, 1);
+            uint64_t beginWWDeviceIndex = readIndex(payload, 9);
```

Comment:
You seem to have reverted to using `WW` instead of `KW` when renaming these variables.